### PR TITLE
Log errors to gcs

### PR DESCRIPTION
**Pull request by DannyLee12**
1. Logs messages to GCS. Does not raise an exception, but does log errors so will still be very visible in the logs.

### CODE REVIEW

File: solr-indexer/logging_config.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        formatter = logging.Formatter('%(asctime)s - %(levelname)-7s - [%(name)s:%(funcName)s:%(filename)s] %(module)s: %(message)s')
+        handler.setFormatter(formatter)
+        handler.set_name('StreamHandler')
+
+    logger = logging.getLogger(name)
+    logger.setLevel(logging_level)
+    logger.addHandler(handler)
+
+    # Update the dependent alice connector logs
+    for k, v in logging.Logger.manager.loggerDict.items():
```
Comment (DannyLee12) [inline solr-indexer/logging_config.py]: `v` is not used. 

Can just use 
`for k in logging.Logger.manager.loggerDict:`

File: solr-indexer/logging_config.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"""Contextually test where application is running and configure logging accordingly"""
+import logging.config
+import os
+
+from google.cloud import logging as gcloud_logging
+from google.cloud.logging.handlers import AppEngineHandler
+
+
+def configure_logger(name: str, logging_level: int):
```
Comment (DannyLee12) [inline solr-indexer/logging_config.py]: `def configure_logger(name: str, logging_level: int) -> logger`?
Everyone knows what a logger is, can you return it with an -> You might need to import it?

File: solr-indexer/solr-indexer.py [hunk 3]
```diff
@@ -59,13 +57,17 @@ def pubsub_push():
                                                                                               dsn,
                                                                                               collection, doc_type))
 
+    # For error logging, pass the row to the global
+    # logging.debug("Solr message to be indexed: {}".format(solr_message))
```
Comment (DannyLee12) [inline solr-indexer/solr-indexer.py]: Remove or uncomment

File: solr-indexer/logging_config.py (hunk 2 above)
Comment (thomasfowler) [inline solr-indexer/logging_config.py]: OK

**Pull request merged**